ros2interface/ros2interface/verb/package.py

```diff
@@ -23,7 +23,7 @@ class PackageVerb(VerbExtension):
     def add_arguments(self, parser, cli_name):
         arg = parser.add_argument(
             'package_name',
-            help="Name of the ROS package (e.g. 'std_msgs, std_srvs, etc.')")
+            help='Name of the ROS package (e.g. "example_interfaces")')
```

Comment:
Please keep the single quote for the user to see in the help message.

Same below.

Reply:
Do you mean to swap the single quotes and double quotes? I only made the change to this line for consistency with the other help message strings.

Reply:
> Do you mean to swap the single quotes and double quotes?

Yes, restore to the state it was before the patch.

> I only made the change to this line for consistency with the other help message strings.

I don't see any other help messages with quotes in the touched files so I assume you are referring to some other files. Without knowing which ones I would guess they should be updated for consistency instead.

Reply:
The other two help messages I touched in the PR have the same pattern: double quotes inside single quotes.

Reply:
Oh, I didn't notice that. Afaik we use show single quotes in user visible message in almost all the code base.

Reply:
I can make the inverse change (or no change at all since it not very related to this PR) if you prefer.

Reply:
I see 8 cases where double quotes are shown to users in Python argument help strings vs. 65 showing single quotes to users (based on a quick grep). So I would suggest using the later.

Reply:
PRs for the two repos with diverging quotes:
* ros2/demos#447
* ros2/rosbag2#416

Reply:
I created a separate PR for ros2cli: https://github.com/ros2/ros2cli/pull/517/

I can address any conflicts with this PR.

Reply:
rebased with #517 merged.